{"uuid":"4a461a32-7337-4990-aaf0-03bbe234c140","historyId":"8bf9408689cde93e0668e657e595ffe7","fullName":"Array: The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits","labels":[{"name":"tag","value":"arrays"},{"name":"host","value":"LAPTOP-RSES787T"},{"name":"thread","value":"59532@LAPTOP-RSES787T.main(1)"},{"name":"feature","value":"Array"},{"name":"story","value":"The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits"},{"name":"suite","value":"Array"},{"name":"testClass","value":"The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/5arrays.feature"},{"name":"package","value":"src.test.resources.features.5arrays_feature.Array"}],"links":[],"name":"The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits","status":"passed","stage":"finished","steps":[{"name":"Given  The user is on \"Question\" page of \"Find Numbers with Even Number of Digits\" after logged in","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673417480160,"stop":1673417480295},{"name":"When  The user enter python code with invalid syntax in tryEditor from sheet \"pythonCode\" and 1","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673417480295,"stop":1673417480684},{"name":"And  The user clicks on run button","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673417480684,"stop":1673417480863},{"name":"Then  The user should be presented with error message as \"SyntaxError: bad input on line 1\"","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673417480863,"stop":1673417480875}],"attachments":[],"parameters":[{"name":"Sheetname","value":"pythonCode"},{"name":"RowNumber","value":"1"}],"start":1673417480159,"stop":1673417480875}